privacy/scrubber.go

```diff
@@ -68,9 +74,11 @@ func (scrubber) ScrubDevice(device *openrtb.Device, ipv6 ScrubStrategyIPV6, geo
 	deviceCopy.DIDSHA1 = ""
 	deviceCopy.DPIDMD5 = ""
 	deviceCopy.DPIDSHA1 = ""
-	deviceCopy.IFA = ""
-	deviceCopy.MACMD5 = ""
-	deviceCopy.MACSHA1 = ""
+	if macAndIFA {
```

Comment:
This is actually not related to AMP. We were only removing these fields for COPPA, but after consolation with our privacy advisors we are extending it to all privacy policies (COPPA, GDPR, CCPA). 

 Can we leave this logic as it was before please?

Reply:
The logic currently is we remove these unless the AMP flag is set in the config and the request is AMP. The idea is that the old AMP behavior is preserved for AMP requests, but the new behavior is preserved otherwise.

Reply:
reverted